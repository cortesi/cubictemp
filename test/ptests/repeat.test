<!--(for this in range(3))-->
    one
<!--(end)-->

<!--(for this in range(3))
    two
(end)-->

<!--(for this in range(3))-->  
    three
<!--(end)-->

<!--(for this in range(3))
    four
(end)-->

<!--(for this in range(3))
    @!this!@
(end)-->

<!--(for this in range(3))
    @!this!@@!this!@
(end)-->
