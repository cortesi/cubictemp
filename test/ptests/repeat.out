one
one
one

two
two
two

three
three
three

four
four
four

0
1
2

00
11
22
